docs/src/main/asciidoc/spring-cloud-zookeeper.adoc

```diff
@@ -122,9 +123,10 @@ public void registerThings() {
 
 Netflix Eureka supports having instances registered with the server that are `OUT_OF_SERVICE` and not returned as active service instances. This is very useful for behaviors such as blue/green deployments. The Curator Service Discovery recipe does not support this behavior. Taking advantage of the flexible payload has let Spring Cloud Zookeeper implement `OUT_OF_SERVICE` by updating some specific metadata and then filtering on that metadata in the Ribbon `ZookeeperServerList`. The `ZookeeperServerList` filters out all non-null instance statuses that do not equal `UP`. If the instance status field is empty, it is considered `UP` for backwards compatibility. To change the status of an instance POST `OUT_OF_SERVICE` to the `ServiceRegistry` instance status actuator endpoint.
 
- ----
- $ echo -n OUT_OF_SERVICE | http POST http://localhost:8081/service-registry/instance-status
- ----
+[source,sh,indent=0]
+----
+$ http POST http://localhost:8081/service-registry status=OUT_OF_SERVICE overriddenStatus=OUT_OF_SERVICE
```

Comment:
so, `overriddenStatus` is specific to s-c-netflix. s-c-zk only has `status`.

Reply:
Okay, will commit the change this evening!